app/views/components/_banner.html.erb

```diff
@@ -12,23 +11,19 @@
     <%= text %>
   </p>
 <% end %>
-
-<% if aside %>
- <div class="app-c-banner app-c-banner--grid">
-  <div class="grid-row">
-    <div class="column-third">
-      <div class="app-c-banner__aside">
-        <p><%= aside %></p>
+<section class="app-c-banner<% if aside %> app-c-banner--grid<% end %>" aria-label="Notice">
+  <% if aside %>
+    <div class="grid-row">
+      <div class="column-third">
```

Comment:
In this case the grid classes would be better suited to being `.govuk-o-grid__row` `.govuk-o-grid__column` etc. Since objects are shared across components.